reverse_engineering/helpers/bigQueryHelper.js

```diff
@@ -5,6 +5,19 @@ const createBigQueryHelper = (client, log) => {
 		return datasets;
 	};
 
+	const getRequiredDatasets = async (datasetId) => {
+		if (!datasetId) {
+			return getDatasets()
+		}
+
+		try {
+			const dataset = await getDataset(datasetId)
+			return [dataset]
+		} catch (err) {
+			return getDatasets()
```

Comment:
Better to throw an error. Getting all database names leads to non-obvious behavior and skipping the real error.